airbyte-cron/src/main/resources/application-control.yml

```diff
@@ -0,0 +1,40 @@
+datasources:
```

Comment:
We shouldn't need this file in the cron service.

Reply:
The issue here is that all the Beans are being initiated because the cron module depends on the worker one which has all the beans being declared.

Reply:
The eager initiation was set to true, I am removing it and will try to clean this list. Unfortunately we will need more config than what we thought.

Reply:
Yeah...this is because we depend on `airbyte-workers`.  That causes Micronaut to find all singletons.  The solution, ultimately, is to extract the shared code to a library module that both can depend on.

Reply:
We sync up with @gosusnp and discover that we have ApplicationInitializer. I think that the best way to do it is to have the cron module to not depends on the worker module. I will try to extract the temporal client from it in another PR.

Reply:
@benmoriceau Agreed...that is the best course here.  It will cause some conflict with the airbyte-workers Micronaut PR, but if you are able to extract and get that merged first, I can easily update the PR.

Reply:
@jdpgrailsdev After merging and merging the extraction of the TemporalClient, I'll clean this file.